drop-in/src/main/java/com/adyen/checkout/dropin/ui/DropInActivity.kt

```diff
@@ -173,18 +173,7 @@ class DropInActivity :
         }
 
         if (noDialogPresent()) {
-            when {
-                dropInViewModel.shouldSkipToSinglePaymentMethod() -> {
-                    val firstPaymentMethod = dropInViewModel.paymentMethodsApiResponse.paymentMethods?.firstOrNull()
-                    if (firstPaymentMethod != null) {
-                        showComponentDialog(firstPaymentMethod)
-                    } else {
-                        throw CheckoutException("First payment method is null")
-                    }
-                }
-                dropInViewModel.showPreselectedStored -> showPreselectedDialog()
-                else -> showPaymentMethodsDialog()
-            }
+            dropInViewModel.activityCreated()
```

Comment:
`onCreated` would be a better name imo. That way it doesn't matter if the view model is used by an activity or fragment and it follows the android naming (e.g. onThis, onThat, onWhateverHappened, etc)

Reply:
`onViewCreated` maybe? 🤔

Reply:
Hmm that kinda implies fragment, what do you think about `onReady`?

Reply:
I went with  `onCreated` eventually